app/src/main/kotlin/com/intive/picover/common/bottomsheet/PicoverModalBottomSheet.kt

```diff
@@ -0,0 +1,28 @@
+package com.intive.picover.common.bottomsheet
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.navigationBarsPadding
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ModalBottomSheet
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun PicoverModalBottomSheet(
+	onDismissRequest: () -> Unit,
+
+	content: @Composable (ColumnScope.(bottomSheetModifier: Modifier) -> Unit),
```

Comment:
Maybe instead of passing `Modifier` we can pass here `PaddingValues` and then apply padding on the call side? Like it is in the `Scaffold` implementation – `content: @Composable (PaddingValues) -> Unit`. Just thinking, you can check it.

Reply:
@BorysChajdas If I understand you correctly I already tried it and it did not work. I am adding Modifier to use .navigationBarsPadding()